test/lib/ufe/testAttribute.py

```diff
@@ -17,6 +17,7 @@
 #
 
 from ufeTestUtils import usdUtils, mayaUtils
+from ufeTestUtils.testUtils import assertVectorAlmostEqual
```

Comment:
When adding the Object3d interface test, had to use assertVectorAlmostEqual, which cause me to go beyond the copy-paste threshold of pain, so I factored it out into a utility, and deployed the utility in all tests that use assertVectorAlmostEqual.

Reply:
Great thanks.